### PR TITLE
liboqs/master -> liboqs/main and refactoring.

### PR DESCRIPTION
**Pull request by xvzcf**

### CODE REVIEW

File: httpd/httpd-conf/httpd-ssl.conf [hunk 1]
```diff
@@ -54,7 +54,7 @@ SSLProxyCipherSuite HIGH:MEDIUM:!MD5:!RC4:!3DES
 
 # You might want to select a subset of supported KEMs from https://github.com/open-quantum-safe/liboqs#supported-algorithms
 # Example:
-# SSLOpenSSLConfCmd Curves oqs_kem_default:frodo640aes:frodo1344shake:kyber512:kyber768:kyber1024:newhope512cca:newhope1024cca:ntru_hps2048509:ntru_hps2048677:ntru_hrss701:lightsaber:saber:sidhp434:sidhp503:sikep751
+# SSLOpenSSLConfCmd Curves oqs_kem_default:kyber512:kyber768:kyber1024::lightsaber:saber
```
Comment (baentsch) [inline httpd/httpd-conf/httpd-ssl.conf:57]: a colon too many?

File: nginx/Dockerfile [hunk 2]
```diff
@@ -31,20 +31,20 @@ ARG MAKE_DEFINES
...
-            libtool automake autoconf cmake \
+            libtool automake autoconf cmake ninja \
             make \
             openssl openssl-dev \
             git wget pcre-dev
 
 # get OQS sources
 WORKDIR /opt
-RUN git clone --depth 1 --branch master https://github.com/open-quantum-safe/liboqs && \
+RUN git clone --depth 1 --branch more-downstream-ci https://github.com/open-quantum-safe/liboqs && \
```
Comment (baentsch) [inline nginx/Dockerfile:41]: why not "main"?

File: .circleci/config.yml [hunk 3]
```diff
@@ -29,26 +29,25 @@ jobs:
...
           command: echo $DOCKER_PASSWORD | docker login --username $DOCKER_LOGIN --password-stdin
-      - run:
-          name: Checkout dependencies
-          command: git clone --depth 1 https://github.com/open-quantum-safe/liboqs.git tmp/liboqs && git clone --depth 1 --branch OQS-OpenSSL_1_1_1-stable --single-branch https://github.com/open-quantum-safe/openssl.git tmp/openssl
       - run:
           name: Test Curl
           command: |
-            docker build -t oqs-curl . &&
-            docker build --target dev -t oqs-curl-dev . &&
+            # The CircleCI executor offers 35 cores, but using
```
Comment (baentsch) [inline .circleci/config.yml:35]: Great idea: Thanks!

File: httpd/httpd-conf/httpd-ssl.conf (hunk 1 above)
Comment (xvzcf) [reply httpd/httpd-conf/httpd-ssl.conf:57]: Nice catch.

File: nginx/Dockerfile (hunk 2 above)
Comment (xvzcf) [reply nginx/Dockerfile:41]: This was an accident, will fix.

**Comment by baentsch** [comment]
@xvzcf This should both fix the reference to a non-configured algorithm as well as the underlying test result error (test check basically, which I originally caused by glossing over an "uncooperative" openssl s_test exit code: always 1). If OK please switch to Ready for Review and I'll take a final look.

**Pull request merged**